### ISSUE TITLE
Add missing calendar icon

### ISSUE THREAD

**Issue by ptbrowne** [cozy/cozy-ui#474]
```
<svg width="16" height="16" xmlns="http://www.w3.org/2000/svg"><path d="M13 1h3v2H0V1h3V0h2v1h6V0h2v1zM0 4h16v10.998A1 1 0 0 1 15 16H1c-.552 0-1-.456-1-1.002V4zm2 2v8h12V6H2zm6 3h5v4H8V9z" fill="#95999D" fill-rule="nonzero"/></svg>
```

`icon-calendar`

**Comment by GoOz** [comment]
I don't get this issue. Hasn't it been done already?

**Comment by ptbrowne** [comment]
fixed in #476 . It had not been fixed when the PR was created.
